proto/attest.proto

```diff
@@ -149,6 +149,10 @@ message SecureBootState {
   // Authority events post-separator. Pre-separator authorities
   // are currently not supported.
   Database authority = 4;
+  // Platform Keys
```

Comment:
nit: please keep comments consistent:
"The Secure Boot Platform key, used to sign key exchange keys."